### PR TITLE
Check plotting file

### PR DESCRIPTION
**Pull request by ezve**
This is to check why we see the discrepancy in the imposed vertical velocity as it is shown in the following figure.
v

### CODE REVIEW

**Comment by cekees** [comment]
Looking at the input files here I don't see any gauges set, nor does it use sponge layers. The v component of the velocity boundary condition is set to zero rather than wave_velocity_v. Is it possible the branch is out of date? I will look at the Dingemans problem for now, since it shows the same thing.

**Comment by ezve** [comment]
@cekees  This is the time signal of the vertical velocity at the left boundary extracted from paraview and gauge functions together with the analytical solution. The results correspond to the updated version of the case that I uploaded earlier today. The paraview signal matches with this of the gauge function as expected.
v_vel

File: 2d/linear_waves_flat/TimeSeriesPlot_v5.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+if Datasource!=3:
+
+ NumberOfCases=1
+ x_loc=[]
+ for i in range(0,int(L[0]/gauge_dx+1)): #+1 only if gauge_dx is an exact 
+   x_loc.append(gauge_dx*i)
+
+ NumLocations=int(len(x_loc))
+ NumCol=3*NumLocations +1
+ y_loc=-0.5 #y=0 in the mean free surface
```
Comment (cekees) [inline 2d/linear_waves_flat/TimeSeriesPlot_v5.py]: @ezve and @adimako , it looks to me like this is the problem. The point gauges are located at `y=0.5` and `y=-0.5` is underneath the tank.

**Comment by ezve** [comment]
@cekees and @adimako  just to confirm that @cekees is right. It was my wrong interpretation on were y=0 was defined and below is shown comparison with the correction. Not a big problem after all! 
v_vel

**Comment by cekees** [comment]
We should delete this branch, correct?

**Comment by ezve** [comment]
@cekees I agree

**Pull request closed without merging**